--- old/src/cache.c	2020-08-03 11:31:52.000000000 +0200
+++ new/src/cache.c	2021-07-14 20:34:45.649309242 +0200
@@ -1018,8 +1018,9 @@
         return ENOENT;
 
     if (is_dir(*nodep)) {
-        if (!(*nodep)->utime)
-            update_directory(*nodep, retry);
+      	 // Disable updating directory listing at lookup as this causes high loads on the server.
+        // if (!(*nodep)->utime)
+        //    update_directory(*nodep, retry);
     } else if (is_open(*nodep)) {
         attr_from_cache_file(*nodep);
     }
